frontend/packages/app/models/daemon.ts

```diff
@@ -26,7 +26,7 @@ function queryDaemonInfo(
       }
       return null
     },
-    refetchInterval: 1500,
+    refetchInterval: 30_000,
```

Comment:
this seems like a super slow loop to see when the daemon comes online. I think a lot of the app is disabled until daemon info is ready

I don't understand why all these timers are being slowed down. These are local requests, super fast! Surely that would not be the cause of bad performance? Maybe we need to fix the rendering that changes as a result of these timers

Reply:
I guess this slowness would only affect the initial load, right? Then we may need to poll more quickly. And indeed this call is definitely cheap.

After the initial load, I don't see why we need to do this polling here. This call basically only returns the account and device IDs of the node, and there seem to be no point at all to calling it repeatedly after the initial load.

Reply:
this call will trigger a re-render of the whole app. is either this of refactor all the components to make sure they don't re-render unnecessarily. I'm ok with both

Reply:
I changed this to 10_000 interval